src/webviews/webview-side/ipywidgets/common/scriptManager.ts

```diff
@@ -130,7 +130,9 @@ export class ScriptManager extends EventEmitter {
                         moduleVersion,
                         new Error(`Timeout getting source for ${moduleName}:${moduleVersion}`),
                         true
-                    ).ignoreErrors();
+                    ).catch(() => {
```

Comment:
During dev this fails, as we aren't importing the file that contains the necessary extensions.